syntax = "proto3";

package vnpt.audio;

option cc_enable_arenas = true;

enum AudioEncoding {

  ENCODING_UNSPECIFIED = 0;

  LINEAR_PCM = 1;

  FLAC = 2;

  MULAW = 3;

  ALAW = 20;
}
